test/aircraft/Pilot/clearedAsFiled.spec.js

```diff
@@ -20,7 +20,8 @@ ava.afterEach(() => {
     resetNavigationLibraryFixture();
 });
 
-ava('.clearedAsFiled() grants pilot departure clearance and returns the correct response strings', (t) => {
+// TODO: Needs AircraftModel fixture, not just pilot
```

Comment:
I think one of those exists already?

Reply:
It does. Due to some conflicts, i'd like to handle this test in #897 though if that's alright